### DISCUSSION TITLE
Configurar fa-icons no projeto

### DISCUSSION THREAD

**Discussion by dycorrales** [po-ui/po-angular#917] [Tire suas dúvidas] [answered]
Como instalo o fa-icons no projeto para poder colocar um fa-icon no button

**Comment by jhonyeduardo** [comment, marked as answer]
Você pode utiliza-lo como componente Angular ou como classe css, veja:

- Componente Angular:  https://github.com/FortAwesome/angular-fontawesome
- Fonte CSS:
1. Adicionar o trecho abaixo no index.html da sua aplicação.
```
    <link
      rel="stylesheet"
      href="https://use.fontawesome.com/releases/v5.3.1/css/all.css"
      integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU"
      crossorigin="anonymous"
    />
```
2. Usar o icone desejado no botão:
```
<po-button p-label="Botão" p-icon="fa fa-podcast"></po-button> 
```

**Reply by dycorrales** [reply]
Muito obrigado, vlw
